contracts/modules/capital/CowSwapOperator.sol

```diff
@@ -59,18 +70,34 @@ contract CowSwapOperator {
 
   receive() external payable {}
 
+  /**
+   * @dev Compute the digest of an order using CoW protocol's logic
+   * @param order The order
+   * @return The digest
+   */
   function getDigest(GPv2Order.Data calldata order) public view returns (bytes32) {
     bytes32 hash = GPv2Order.hash(order, domainSeparator);
     return hash;
   }
 
+  /**
+   * @dev Compute the UID of an order using CoW protocol's logic
+   * @param order The order
+   * @return The UID (56 bytes)
+   */
   function getUID(GPv2Order.Data calldata order) public view returns (bytes memory) {
     bytes memory uid = new bytes(56);
     bytes32 digest = getDigest(order);
     GPv2Order.packOrderUidParams(uid, digest, order.receiver, order.validTo);
     return uid;
   }
 
+  /**
+   * @dev Approve a given order to be executed, by presigning it on CoW protocol's settlement contract
+   * @dev Only one order can be open at the same time
```

Comment:
I think the second `@dev` is not needed. It can be just `*  Only one order can be open at the same time`. I think we should also add a comment that one the assets should be ETH